common/ethnicities/wc_ethnicities_highelf_rework.txt

```diff
@@ -0,0 +1,472 @@
+﻿@beauty1min = 0.35
+@beauty1max = 0.65
+
+@beauty2min = 0.4
+@beauty2max = 0.6
+
+@beauty3min = 0.45
+@beauty3max = 0.55
+
+# Warcraft
+###Skin colors
+@human_skin_color_x_min = 0.15
+@human_skin_color_x_max = 0.2
+
+###Hair colors
+@blonde_color_x_min = 0.2
+@blonde_color_x_max = 0.25
+@blonde_color_y_min = 0.15
+@blonde_color_y_max = 0.62
+
+@brown_color_x_min = 0.08
+@brown_color_x_max = 0.2
+@brown_color_y_min = 0.86
+@brown_color_y_max = 0.95
+
+@red_color_x_min = 0.05
+@red_color_x_max = @brown_color_x_min
+@red_color_y_min = 0.53
+@red_color_y_max = 0.65
+
+@white_color_y_min = 0.05
+@white_color_y_max = 0.05
+
+###Eye colors
+@gold_color_x_min = 0.25
+@gold_color_x_max = 0.31
+@gold_color_y_min = 0.34
+@gold_color_y_max = 0.62
+
+@blue_eye_color_x_min = 0.6
+@blue_eye_color_x_max = 0.7
+
+@arcane_color_y_min = 0.34
+@arcane_color_y_max = 0.53
+
+@green_color_x_min = 0.33
+
+@elven_red_color_x_max = 1
+
+@blue_color_x_min = @blue_eye_color_x_min
+@blue_color_x_max = 0.83
+
+highelf_rework = {
```

Comment:
So, the highborne on the latest arts look a lot like high elves, so what about them using high elves as a base instead of night elves?
<details>
<summary>Click to expand</summary>

https://wow.gamepedia.com/File:Warbringers_Azshara_holding_a_barrier.jpg
https://wow.gamepedia.com/File:Warbringers_elves.jpg

</details>

Reply:
> 
> 
> So, the highborne on the latest arts look a lot like high elves, so what about them using high elves as a base instead of night elves?

Changed in 1e5e464e32fa5e2b7a77346381fbf8c773871852. Also removed the duplicate Highborne entry from `wc_ethnicities_nelf_rework`.

Reply:
Though their ears are kinda different. They're more similar to night elven ones.